yarn.lock

```diff
@@ -16518,6 +16518,11 @@ reselect@^4.0.0, reselect@^4.1.5:
   resolved "https://registry.yarnpkg.com/reselect/-/reselect-4.1.6.tgz#19ca2d3d0b35373a74dc1c98692cdaffb6602656"
   integrity sha512-ZovIuXqto7elwnxyXbBtCPo9YFEr3uJqj2rRbcOOog1bmu2Ag85M4hixSwFWyaBMKXNgvPaJ9OSu9SkBPIeJHQ==
 
+reselect@^4.1.7:
```

Comment:
should we remove the above reselect? 4.1.7 should satisfy `reselect@^4.0.0, reselect@^4.1.5` right?